src/Core/Usecase/Form/SearchFormStats.php

```diff
@@ -37,20 +36,60 @@ public function interact()
         // ... and get the search filters for this entity
         $search = $this->getSearch();
         $search->setFilter('form_id', $this->getIdentifier('form_id'));
+        $search->setFilter('created_after', $this->getIdentifier('created_after'));
```

Comment:
note that this would change if you change the code so that created_after is a filter instead of an identifier